src/devtools/client/debugger/src/reducers/source-actors.ts

```diff
@@ -112,6 +112,10 @@ export default function update(state = initial, action: AnyAction) {
       state.values = clearBreakpointHitCounts(state);
       break;
 
+    case "timeline/setTrimRegion":
+      state.values = clearBreakpointHitCounts(state);
```

Comment:
⚠️ FYI this is actually mutating `state`, because we don't have `createSlice` here and we aren't using Immer at all.  Gotta do a full `{...state, values: clearBreakpointHitCounts(state)}` here.

Reply:
Actually it looks like we also already do this with `"app/setLoadedRegions"` above as well - can you fix that too?

Reply:
> we don't have createSlice here

Oh shoot I totally did not realize this. Thanks for the catch! You've done such a thorough job I just assume that we have `createSlice` everywhere 😅